### ISSUE TITLE
Union with `prettier-php`

### ISSUE THREAD

**Issue by alexander-akait** [chris-l/php-unparser#15]
https://github.com/prettier/plugin-php :star:

**Comment by chris-l** [comment]
Hi, I'm not sure what this issue is about?
You want to use php-unparser on your prettier plugin?

**Comment by alexander-akait** [comment]
@chris-l move best pretty practice in prettier-php :+1:

**Comment by chris-l** [comment]
I'm sorry, I'm not sure if I understand correctly.
I'm guessing that by "pretty practice", you are referring to the pretty printer I created as demo. And you want to use that demo on your prettier-php repo. Is that correct?

By pretty printer demo, I mean this: https://chris-l.github.io/php-unparser/

Well, I'm only using glayzzle's php-parser and my php-unparser for that, so you can just import both php-parser and php-unparser to create a pretty printer.

**Comment by alexander-akait** [comment]
@chris-l yes, you have experience with some pretty output using `php-parser` (as link above), i am glad if you help us (`prettier-php`) to improve pretty print using you experience here.
